.github/workflows/ci.yml

```diff
@@ -65,6 +65,9 @@ jobs:
         gem:
           - opentelemetry-exporter-jaeger
           - opentelemetry-exporter-otlp
+          - opentelemetry-exporter-otlp-common
+          - opentelemetry-exporter-otlp-grpc
```

Comment:
@arielvalentin or @ahayworth oh keepers of ci, what's the right way to skip jruby for this specific gem?

Reply:
I see a stanza for the instrumentation gems that allows skipping jruby, should I mimic that?

Reply:
You are all correct. There are ways to DRY it out but I'd rather not, because I think we should continue to evolve the CI system in a slightly different way.

I would copy and paste your way to victory here, and I can help out if you need.